ldapadmin/src/main/java/org/georchestra/ldapadmin/ds/AccountDaoImpl.java

```diff
@@ -446,7 +447,16 @@ private void setAccountField(DirContextOperations context, String fieldName, Obj
 
         if (!isNullValue(value)) {
             context.setAttributeValue(fieldName, value);
-        }
+        } else {
+            Object[] values = context.getObjectAttributes(fieldName);
+            if(values != null){
+            if (values.length == 1) {
+                LOG.info("Removing attribue " + fieldName);
+                context.removeAttributeValue(fieldName, values[0]);
+            } else {
+                LOG.error("Multiple values encountered for field " + fieldName +", expected a single value");
```

Comment:
It can, multiple values for a field can be added using another ldap client (e.g. apache directory studio), and the ldap schema does not disallow this.